src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplay.cs

```diff
@@ -127,26 +156,50 @@ public static class SymbolDisplay
             int position,
             SymbolDisplayFormat format = null)
         {
-            format = format ?? SymbolDisplayFormat.MinimallyQualifiedFormat;
-            return ToDisplayParts(symbol, topLevelNullabilityOpt: null, semanticModel, position, format, minimal: true);
+            format ??= SymbolDisplayFormat.MinimallyQualifiedFormat;
+            return ToDisplayParts(symbol, nullableAnnotationOpt: null, semanticModel, position, format, minimal: true);
         }
 
 #pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
         public static ImmutableArray<SymbolDisplayPart> ToMinimalDisplayParts(
             ITypeSymbol symbol,
-            CodeAnalysis.NullableFlowState topLevelNullability,
+            CodeAnalysis.NullableFlowState nullableFlowState,
+            SemanticModel semanticModel,
+            int position,
+            SymbolDisplayFormat format = null)
+        {
+            format ??= SymbolDisplayFormat.MinimallyQualifiedFormat;
+            return ToDisplayParts(symbol, nullableFlowState, semanticModel, position, format, minimal: true);
+        }
+
+        public static ImmutableArray<SymbolDisplayPart> ToMinimalDisplayParts(
+            ITypeSymbol symbol,
+            CodeAnalysis.NullableAnnotation nullableAnnotation,
             SemanticModel semanticModel,
             int position,
             SymbolDisplayFormat format = null)
+        {
+            format ??= SymbolDisplayFormat.MinimallyQualifiedFormat;
+            return ToDisplayParts(symbol, nullableAnnotation, semanticModel, position, format, minimal: true);
+        }
 #pragma warning restore RS0026 // Do not add multiple public overloads with optional parameters
+
+        private static ImmutableArray<SymbolDisplayPart> ToDisplayParts(
+            ITypeSymbol symbol,
+            CodeAnalysis.NullableFlowState nullableFlowState,
+            SemanticModel semanticModelOpt,
+            int positionOpt,
+            SymbolDisplayFormat format,
+            bool minimal)
         {
-            format = format ?? SymbolDisplayFormat.MinimallyQualifiedFormat;
-            return ToDisplayParts(symbol, topLevelNullability, semanticModel, position, format, minimal: true);
+            // PROTOTYPE(nullable-api): Refactor this. We need to be able to handle non-TypeSymbol inputs, and add tests
```

Comment:
> and add tests [](start = 106, length = 14)

Please consider moving this comment to the API, if there's not already a PROTOTYPE comment for testing the API.